[0m11:07:18.841023 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024E9E761510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024E9F7A4D90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024E9F7A4B20>]}


============================== 11:07:18.857142 | 061525ca-af6c-43dc-b9d5-5d06e3ff6652 ==============================
[0m11:07:18.857142 [info ] [MainThread]: Running with dbt=1.10.4
[0m11:07:18.857142 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\patil\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init .', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:08:31.964309 [debug] [MainThread]: Starter project path: C:\Users\patil\anaconda3\envs\dbt-snowflake\lib\site-packages\dbt\include\starter_project
[0m11:08:32.010269 [info ] [MainThread]: 
Your new dbt project "dbtsnowflake" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:08:32.010269 [info ] [MainThread]: Setting up your profile.
[0m11:10:28.973466 [info ] [MainThread]: Profile dbtsnowflake written to C:\Users\patil\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m11:10:28.975474 [debug] [MainThread]: Command `dbt init` succeeded at 11:10:28.975474 after 190.23 seconds
[0m11:10:28.975474 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024E9E761510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024E9F7A4AC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024E9F7A39D0>]}
[0m11:10:28.975474 [debug] [MainThread]: Flushing usage events
[0m11:10:29.923455 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:10:43.735071 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021297D914B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021298DD0DC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021298DD0B50>]}


============================== 11:10:43.735071 | 5f65da0b-ada3-4c03-9fc4-6076eef87d46 ==============================
[0m11:10:43.735071 [info ] [MainThread]: Running with dbt=1.10.4
[0m11:10:43.735071 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\patil\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m11:10:43.761013 [info ] [MainThread]: dbt version: 1.10.4
[0m11:10:43.761013 [info ] [MainThread]: python version: 3.10.18
[0m11:10:43.775699 [info ] [MainThread]: python path: C:\Users\patil\anaconda3\envs\dbt-snowflake\python.exe
[0m11:10:43.776734 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m11:10:45.056479 [info ] [MainThread]: Using profiles dir at C:\Users\patil\.dbt
[0m11:10:45.056479 [info ] [MainThread]: Using profiles.yml file at C:\Users\patil\.dbt\profiles.yml
[0m11:10:45.056479 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\patil\VS_Code\dbt-snowflake\dbt_project.yml
[0m11:10:45.056479 [info ] [MainThread]: adapter type: snowflake
[0m11:10:45.056479 [info ] [MainThread]: adapter version: 1.9.0
[0m11:10:45.056479 [info ] [MainThread]: Configuration:
[0m11:10:45.069257 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m11:10:45.071343 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m11:10:45.072345 [info ] [MainThread]: Required dependencies:
[0m11:10:45.074008 [debug] [MainThread]: Executing "git --help"
[0m11:10:45.128748 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m11:10:45.129749 [debug] [MainThread]: STDERR: "b''"
[0m11:10:45.130747 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m11:10:45.131753 [info ] [MainThread]: Connection:
[0m11:10:45.133130 [info ] [MainThread]:   account: onvtzee-web86246
[0m11:10:45.133130 [info ] [MainThread]:   user: shantanu
[0m11:10:45.133130 [info ] [MainThread]:   database: SHANTANU_DB
[0m11:10:45.133130 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m11:10:45.133130 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m11:10:45.138862 [info ] [MainThread]:   schema: PUBLIC
[0m11:10:45.139872 [info ] [MainThread]:   authenticator: None
[0m11:10:45.140873 [info ] [MainThread]:   oauth_client_id: None
[0m11:10:45.142348 [info ] [MainThread]:   query_tag: None
[0m11:10:45.143360 [info ] [MainThread]:   client_session_keep_alive: False
[0m11:10:45.143947 [info ] [MainThread]:   host: None
[0m11:10:45.143947 [info ] [MainThread]:   port: None
[0m11:10:45.147521 [info ] [MainThread]:   proxy_host: None
[0m11:10:45.147521 [info ] [MainThread]:   proxy_port: None
[0m11:10:45.150745 [info ] [MainThread]:   protocol: None
[0m11:10:45.151344 [info ] [MainThread]:   connect_retries: 1
[0m11:10:45.152398 [info ] [MainThread]:   connect_timeout: None
[0m11:10:45.153353 [info ] [MainThread]:   retry_on_database_errors: False
[0m11:10:45.154398 [info ] [MainThread]:   retry_all: False
[0m11:10:45.155351 [info ] [MainThread]:   insecure_mode: False
[0m11:10:45.156956 [info ] [MainThread]:   reuse_connections: True
[0m11:10:45.158965 [info ] [MainThread]: Registered adapter: snowflake=1.9.0
[0m11:10:45.716516 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m11:10:46.093272 [debug] [MainThread]: Using snowflake connection "debug"
[0m11:10:46.093272 [debug] [MainThread]: On debug: select 1 as id
[0m11:10:46.093272 [debug] [MainThread]: Opening a new connection, currently in state init
[0m11:10:48.865600 [debug] [MainThread]: SQL status: SUCCESS 1 in 2.761 seconds
[0m11:10:48.866382 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m11:10:48.866382 [info ] [MainThread]: [31m1 check failed:[0m
[0m11:10:48.866382 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\patil\VS_Code\dbt-snowflake\dbt_project.yml> not found

[0m11:10:48.870292 [debug] [MainThread]: Command `dbt debug` failed at 11:10:48.870292 after 5.28 seconds
[0m11:10:48.870292 [debug] [MainThread]: Connection 'debug' was left open.
[0m11:10:48.870292 [debug] [MainThread]: On debug: Close
[0m11:10:49.474969 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021297D914B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002129C4F3BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002129C463A90>]}
[0m11:10:49.474969 [debug] [MainThread]: Flushing usage events
[0m11:10:50.414448 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m17:27:26.418510 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002542FAD1510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025430B14D60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025430B14B80>]}


============================== 17:27:26.424919 | c456d009-6d2a-41e5-a95e-3964ad60603c ==============================
[0m17:27:26.424919 [info ] [MainThread]: Running with dbt=1.10.4
[0m17:27:26.425919 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\patil\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True'}
[0m17:27:53.833263 [info ] [MainThread]: A project called dbtsnowflake already exists here.
[0m17:27:53.846931 [debug] [MainThread]: Command `dbt init` succeeded at 17:27:53.835290 after 27.52 seconds
[0m17:27:53.847950 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002542FAD1510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025430B15420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025430B14D60>]}
[0m17:27:53.848946 [debug] [MainThread]: Flushing usage events
[0m17:27:54.849209 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m17:28:05.320329 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000191B6E01480>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000191B7E44CD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000191B7E44AF0>]}


============================== 17:28:05.325476 | c661fc28-715a-4f27-921d-7f7da3048175 ==============================
[0m17:28:05.325476 [info ] [MainThread]: Running with dbt=1.10.4
[0m17:28:05.326470 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\patil\\.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m17:28:19.674934 [debug] [MainThread]: Starter project path: C:\Users\patil\anaconda3\envs\dbt-snowflake\lib\site-packages\dbt\include\starter_project
[0m17:28:19.720796 [info ] [MainThread]: 
Your new dbt project "shantanudbtsnowflake" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:28:19.721799 [info ] [MainThread]: Setting up your profile.
[0m17:40:32.202878 [info ] [MainThread]: Profile shantanudbtsnowflake written to C:\Users\patil\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m17:40:32.206876 [debug] [MainThread]: Command `dbt init` succeeded at 17:40:32.205878 after 746.99 seconds
[0m17:40:32.207877 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000191B6E01480>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000191B7DDF8E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000191B7E42F50>]}
[0m17:40:32.208877 [debug] [MainThread]: Flushing usage events
[0m17:40:33.130610 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:08:49.011439 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000164890514E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001648A098EB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001648A098CD0>]}


============================== 10:08:49.017438 | 1f7d7485-a5a2-4a60-a9fa-e5621889a2dc ==============================
[0m10:08:49.017438 [info ] [MainThread]: Running with dbt=1.10.4
[0m10:08:49.018438 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\patil\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:08:49.020607 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\patil\VS_Code\dbt-snowflake\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m10:08:49.022217 [debug] [MainThread]: Command `dbt run` failed at 10:08:49.022217 after 0.12 seconds
[0m10:08:49.023219 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000164890514E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001648A098FA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001648A098F70>]}
[0m10:08:49.025219 [debug] [MainThread]: Flushing usage events
[0m10:08:49.935169 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:19:30.627983 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015F672214E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015F68268EB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015F68268CD0>]}


============================== 10:19:30.633898 | 940b85be-9d0a-4200-8dc7-9f260e30c338 ==============================
[0m10:19:30.633898 [info ] [MainThread]: Running with dbt=1.10.4
[0m10:19:30.635865 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\patil\\.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m10:19:30.636899 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\patil\VS_Code\dbt-snowflake\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m10:19:30.637862 [debug] [MainThread]: Command `dbt run` failed at 10:19:30.637862 after 0.11 seconds
[0m10:19:30.638862 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015F672214E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015F68268FA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015F68268F70>]}
[0m10:19:30.639861 [debug] [MainThread]: Flushing usage events
[0m10:19:31.531424 [debug] [MainThread]: An error was encountered while trying to flush usage events
